nats/src/lib.rs

```diff
@@ -13,6 +13,9 @@
 
 //! A Rust client for the NATS.io ecosystem.
 //!
+//! <div class="warning"> This is the old legacy client. Will not get new features or updates beyond critical security fixes.
+//! Consider using <a href="https://crates.io/crates/async-nats">async-nats</a> instead. </div>
```

Comment:
I'd say "use" instead of "consider" since it's deprecated.
```suggestion
//! Use <a href="https://crates.io/crates/async-nats">async-nats</a> instead. </div>
```